### ISSUE TITLE
Frequent out-of-memory errors when playing audio during card creation

### ISSUE THREAD

**Issue by 0-CAT** [killergerbah/asbplayer#319]
Been using ASB Player for a long time and just recently, the site has started running out of memory when using audio playback to create cards. I don't recall it doing this even once until just recently.

![image](https://github.com/killergerbah/asbplayer/assets/104591838/32b81f22-eeb8-42a8-bb38-7240bcfd08f3)

After opening a fresh tab with a video file, I was able to get the tab to rocket up to nearly 2 GB of memory consumption just by playing an audio clip.

![asb-memory](https://github.com/killergerbah/asbplayer/assets/104591838/64ffb02e-1102-4151-8b4a-54ba8bcf47d9)

**Comment by 0-CAT** [comment]
> @0-CAT I released a large change to asbplayer today. Part of that included how audio clips and screenshots from local media files are handled. This was probably the reason for the memory problems. I've deployed some speculative fixes for the problem you're seeing. If you can let me know if the problem is still happening.

Yup, something you did in there worked. Stays under 200k when playing audio now. Thank you!

**Comment by killergerbah** [comment]
Thanks for reporting the issue!
